### ISSUE TITLE
Error logs on Samsung j7

### ISSUE THREAD

**Issue by MahdiPishguy** [luanpotter/audioplayers#377]
**Samsung j7:**

maybe this link can be solve this problem:

https://stackoverflow.com/questions/16238218/android-media-player-streaming-issue-on-samsung-devices

```
2020-01-01 10:07:14.623 3241-3241/? E/SDAgentPackageStateReceiver: Not going to handle 'xyz.luan.audioplayers_example'!
2020-01-01 10:07:15.839 3241-3241/? E/SDAgentPackageStateReceiver: Not going to handle 'xyz.luan.audioplayers_example'!
2020-01-01 10:07:16.642 26914-26914/? E/SPPClientService: [PackageInfoChangeReceiver] [handlePkgRemovedEvent] PackageName : xyz.luan.audioplayers_example, true, false
2020-01-01 10:07:18.666 3241-3332/? E/WindowManager: win=Window{82d42a4 u0 Splash Screen xyz.luan.audioplayers_example EXITING} destroySurfaces: appStopped=false win.mWindowRemovalAllowed=true win.mRemoveOnExit=true win.mViewVisibility=0 caller=com.android.server.wm.AppWindowToken.destroySurfaces:896 com.android.server.wm.AppWindowToken.destroySurfaces:877 com.android.server.wm.WindowState.onExitAnimationDone:5473 com.android.server.wm.WindowStateAnimator.onAnimationFinished:320 com.android.server.wm.WindowState.onAnimationFinished:5895 com.android.server.wm.-$$Lambda$yVRF8YoeNdTa8GR1wDStVsHu8xM.run:2 com.android.server.wm.SurfaceAnimator.lambda$getFinishedCallback$0:100 
2020-01-01 10:08:08.416 9224-9252/xyz.luan.audioplayers_example E/MediaPlayerNative: error (1, -2147483648)
2020-01-01 10:08:08.417 9224-9224/xyz.luan.audioplayers_example E/MediaPlayer: Error (1,-2147483648)
2020-01-01 10:08:08.418 9224-9224/xyz.luan.audioplayers_example E/MediaPlayerNative: stop called in state 0, mPlayer(0xcc3fe6a0)
2020-01-01 10:08:08.418 9224-9224/xyz.luan.audioplayers_example E/MediaPlayerNative: error (-38, 0)
2020-01-01 10:09:15.843 9224-9452/xyz.luan.audioplayers_example E/MediaPlayerNative: error (1, -2147483648)
2020-01-01 10:09:15.843 9224-9224/xyz.luan.audioplayers_example E/MediaPlayer: Error (1,-2147483648)
2020-01-01 10:09:15.844 9224-9224/xyz.luan.audioplayers_example E/MediaPlayerNative: stop called in state 0, mPlayer(0xcd338b20)
2020-01-01 10:09:15.845 9224-9224/xyz.luan.audioplayers_example E/MediaPlayerNative: error (-38, 0)
2020-01-01 10:10:56.100 9224-9224/xyz.luan.audioplayers_example E/MediaPlayerNative: stop called in state 4, mPlayer(0xd057a240)
2020-01-01 10:10:56.100 9224-9224/xyz.luan.audioplayers_example E/MediaPlayerNative: error (-38, 0)
2020-01-01 10:11:39.723 3241-3241/? E/SDAgentPackageStateReceiver: Not going to handle 'xyz.luan.audioplayers_example'!
2020-01-01 10:11:40.052 26914-26914/? E/SPPClientService: [PackageInfoChangeReceiver] [handlePkgRemovedEvent] PackageName : xyz.luan.audioplayers_example, true, false
2020-01-01 10:11:40.270 3241-3241/? E/SDAgentPackageStateReceiver: Not going to handle 'xyz.luan.audioplayers_example'!
2020-01-01 10:11:43.929 2807-3012/? E/BufferQueueProducer: [Waiting For Debugger: xyz.luan.audioplayers_example#0] disconnect: not connected (req=1)
2020-01-01 10:11:47.361 3241-3332/? E/WindowManager: win=Window{d9804f2 u0 Splash Screen xyz.luan.audioplayers_example EXITING} destroySurfaces: appStopped=false win.mWindowRemovalAllowed=true win.mRemoveOnExit=true win.mViewVisibility=0 caller=com.android.server.wm.AppWindowToken.destroySurfaces:896 com.android.server.wm.AppWindowToken.destroySurfaces:877 com.android.server.wm.WindowState.onExitAnimationDone:5473 com.android.server.wm.WindowStateAnimator.onAnimationFinished:320 com.android.server.wm.WindowState.onAnimationFinished:5895 com.android.server.wm.-$$Lambda$yVRF8YoeNdTa8GR1wDStVsHu8xM.run:2 com.android.server.wm.SurfaceAnimator.lambda$getFinishedCallback$0:100 
2020-01-01 10:12:29.186 10069-10092/xyz.luan.audioplayers_example E/MediaPlayerNative: error (1, -2147483648)
2020-01-01 10:12:29.187 10069-10069/xyz.luan.audioplayers_example E/MediaPlayer: Error (1,-2147483648)
2020-01-01 10:12:29.188 10069-10069/xyz.luan.audioplayers_example E/MediaPlayerNative: stop called in state 0, mPlayer(0xcff04120)
2020-01-01 10:12:29.188 10069-10069/xyz.luan.audioplayers_example E/MediaPlayerNative: error (-38, 0)









2020-01-01 10:18:41.666 10739-10755/xyz.luan.audioplayers_example E/MediaPlayerNative: error (1, -2147483648)
2020-01-01 10:18:41.666 10739-10739/xyz.luan.audioplayers_example E/MediaPlayer: Error (1,-2147483648)
2020-01-01 10:18:41.667 10739-10739/xyz.luan.audioplayers_example E/MediaPlayerNative: stop called in state 0, mPlayer(0xc89f4480)
2020-01-01 10:18:41.667 10739-10739/xyz.luan.audioplayers_example E/MediaPlayerNative: error (-38, 0)
2020-01-01 10:20:01.619 10739-10819/xyz.luan.audioplayers_example A/libc: Fatal signal 11 (SIGSEGV), code 128 (SI_KERNEL), fault addr 0x0 in tid 10819 (players_example), pid 10739 (players_example)
2020-01-01 10:20:01.815 10901-10901/? A/DEBUG: pid: 10739, tid: 10819, name: players_example  >>> xyz.luan.audioplayers_example <<<
2020-01-01 10:20:02.983 3241-3269/? E/WindowManager: RemoteException occurs on reporting focusChanged, w=Window{41ebb49 u0 xyz.luan.audioplayers_example/xyz.luan.audioplayers_example.MainActivity EXITING}
    android.os.DeadObjectException
        at android.os.BinderProxy.transactNative(Native Method)
        at android.os.BinderProxy.transact(Binder.java:1143)
        at android.view.IWindow$Stub$Proxy.windowFocusChanged(IWindow.java:500)
        at com.android.server.wm.WindowState.reportFocusChangedSerialized(WindowState.java:3976)
        at com.android.server.wm.WindowManagerService$H.handleMessage(WindowManagerService.java:5508)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:214)
        at android.os.HandlerThread.run(HandlerThread.java:65)
        at com.android.server.ServiceThread.run(ServiceThread.java:44)
2020-01-01 10:20:02.983 3241-5054/? E/WindowManager: win=Window{41ebb49 u0 xyz.luan.audioplayers_example/xyz.luan.audioplayers_example.MainActivity EXITING} destroySurfaces: appStopped=false win.mWindowRemovalAllowed=true win.mRemoveOnExit=true win.mViewVisibility=0 caller=com.android.server.wm.AppWindowToken.destroySurfaces:896 com.android.server.wm.AppWindowToken.destroySurfaces:877 com.android.server.wm.WindowState.onExitAnimationDone:5473 com.android.server.wm.-$$Lambda$01bPtngJg5AqEoOWfW3rWfV7MH4.accept:2 java.util.ArrayList.forEach:1262 com.android.server.wm.AppWindowToken.onAnimationFinished:2469 com.android.server.wm.AppWindowToken.setVisibility:560 
```

**Comment by luanpotter** [comment]
If it's an issue with the device itself, I don't think there is a lot we can do to fix it...

**Comment by luanpotter** [comment]
I will add it to the troubleshoot guide.

**Comment by luanpotter** [comment]
Closed by #709
